lib/index.js

```diff
@@ -21,49 +21,59 @@ var Slack = module.exports = integration('Slack')
  * Set up our prototype methods.
  */
 
-Slack.prototype.track = send;
-
-/** Send an event to the Slack API. */
-function send(track, fn){
-  /**
-   * Get a human readable name for the user:
-   * 1. Check traits.name
-   * 2. Check traits.firstName + traits.lastName
-   * 3. Check traits.firstName
-   * 4. Check traits.lastName
-   * 5. Check traits.username
-   * 6. Check traits.email
-   * 7. Use 'User ' + userId
-   * 8. Use 'Anonymous user ' + anonymousId
-   *
-   * @return {string}
-   */
-  var getName = function() {
-    var name = track.proxy('context.traits.name');
-    if (name) return name;
-
-    var firstName = track.proxy('context.traits.firstName');
-    var lastName = track.proxy('context.traits.lastName');
-    if (firstName && lastName) return firstName + ' ' + lastName;
-    if (firstName) return firstName;
-    if (lastName) return lastName;
-
-    var username = track.proxy('context.traits.username');
-    if (username) return username;
-
-    var email = track.email();
-    if (email) return email;
-
-    var userId = track.userId();
-    if (userId) return 'User ' + track.userId();
-
-    return 'Anonymous user ' + track.anonymousId();
+Slack.prototype.identify = identify;
+Slack.prototype.track = track;
+
+/**
+ * Identify.
+ */
+
+function identify(identify, fn) {
```

Comment:
Agreed, but that's not a hard rule and this code base is already setup this way so it's not Andy's responsibility to change it.